### PR TITLE
Ci: macos

### LINKED ISSUE #152: CI: Build on macOS

### PR DESCRIPTION
**Pull request by chfast**
Closes #152

### CODE REVIEW

**Comment by chfast** [comment]
Almost works. I don't know what happened to the failing job.

File: circle.yml [hunk 1]
```diff
@@ -291,11 +295,30 @@ jobs:
       - *update-submodules
       - *environment-info
       - *restore-deps-cache
-      - *configure
```
Comment (axic) [inline circle.yml]: Why remove this?

Comment (axic) [reply circle.yml]: I think adding back this will fix the CI error too.

Comment (chfast) [reply circle.yml]: Accident

**Pull request merged**